core/templates/dev/head/domain/exploration/EditableExplorationBackendApiService.js

```diff
@@ -145,12 +155,11 @@ oppia.factory('EditableExplorationBackendApiService', [
 
       /**
        * Deletes an exploration in the backend with the provided exploration
-       * ID. If successful, the exploration will also be deleted from the
-       * ReadOnlyExplorationBackendApiService cache as well. Errors are passed
-       * to the error callback, if one is provided.
+       * ID, for a particular role. If successful, the exploration will
+       * also be deleted from the ReadOnlyExplorationBackendApiService cache
+       * as well. Errors are passed to the error callback, if one is provided.
        */
-      deleteExploration: function(
-        explorationId, role) {
+      deleteExploration: function(explorationId, role) {
```

Comment:
Actually, upon reviewing the code, Role is now handled in the backend and not passed as a variable anymore! So will take out this from backend, and remove associated URL templates.